web/pdf_viewer.css

```diff
@@ -16,6 +16,7 @@
 @import url(text_layer_builder.css);
 @import url(annotation_layer_builder.css);
 @import url(xfa_layer_builder.css);
+@import url(message_bar.css);
```

Comment:
Would this not make more sense to place *before* the `@import url(dialog.css);` line, since that file extends the `.messageBar` class?

Reply:
This wasn't fixed in the correct patch, since the change belongs in `Break messageBar class out`.